src/mutation.cpp

```diff
@@ -501,6 +504,26 @@ void player::activate_mutation( const trait_id &mut )
         print_health();
         tdata.powered = false;
         return;
+    } else if( mut == trait_ROOTS2 ) {
+        double shoe_factor = footwear_factor();
+        if( g->m.has_flag( "PLOWABLE", pos() ) && shoe_factor != 1.0 && !( has_effect( effect_rooted ) ) ) {
+            assign_activity( player_activity( activity_id( "ACT_ROOT" ), MINUTES( 5 ), -1, 0, "Rooting" ),
+                             false );
+        } else if( shoe_factor = 1.0 && !( has_effect( effect_rooted ) ) ) {
```

Comment:
don't you want to compare shoe_factor to something instead of assigning 1.0 to it here?

Reply:
Whoops, should be fixed now.